config/initializers/mas_cms_client.rb

```diff
@@ -0,0 +1,9 @@
+Mas::Cms::Client.config do |c|
+  c.timeout =  ENV['HTTP_REQUEST_TIMEOUT'].to_i
+  c.open_timeout =  ENV['HTTP_REQUEST_TIMEOUT'].to_i
+  c.api_token =  ENV['MAS_CMS_API_TOKEN']
```

Comment:
We don't need the token for fincap. The token is used for post and put requests only. You can remove this line.